pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -254,13 +253,18 @@ export default function HandlerForm({id, endpointId}) {
                           />
                         </div>
                         <div className="col-6">
-                          <GenericInputComponent
-                            type={"text"}
-                            name={"templateType"}
-                            id={"templateTypeInput"}
-                            data={handler && handler.templateType && handler.templateType}
-                            nameOverride={"Template Type"}
-                          />
+                            <SelectInputComponent
+                              options={[
+                                {name: "twig", value: "twig"},
+                                {name: "markdown", value: "markdown"},
+                                {name: "restructuredText", value: "restructuredText"}
+                              ]}
+                              name={"type"}
+                              id={"templateType"}
+                              nameOverride={"Template Type"}
+                              required={true}
+                              data={handler?.templateType}
```

Comment:
Something's going wrong here, the `SelectInputComponent` doesn't actually show the right value (it's always on the first option "twig").